README.md

```diff
@@ -28,9 +29,13 @@ Sample text here.
 
 ## Collaborators
```

Comment:
May remove the list of collaborators altogether, since they will appear in the contributions tab later anyways.